app/js/disease-associations-table-directive.js

```diff
@@ -322,13 +341,14 @@ angular.module('cttvDirectives')
             // legend stuff
             scope.legendText = "Score";
             scope.colors = [];
-            for(var i=0; i<=100; i+=25){
-                var j=i/100;
+            scope.excludedTargetList = attrs.excludedtargetlist;
```

Comment:
This is confusing, you already have excludedTargetList in your scope, why do you need to re-set it with the value in the attrs? (it should be the same)